### PR TITLE
docs: add offline install to 'how-to-build.md'

### PR DESCRIPTION
**Pull request by 123liubao**
Signed-off-by: 123liubao <chenly_61@163.com>

### What this PR does / why we need it:

add offline installation for how-to-build.md
resolve https://github.com/apache/apisix/issues/5785

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by juzhiyuan — APPROVED** [review]
Contents LGTM, just to make sure, the `./apisix` folder is created from where?

**Comment by spacewander** [comment]
@leslie-tsang 
Please take a look.

**Comment by 123liubao** [comment]
> Contents LGTM, just to make sure, the `./apisix` folder is created from where?

@juzhiyuan I'm not sure what do you mean , `sudo mkdir -p apisix` this step is to create the folder.

**Comment by leslie-tsang** [comment]
> Contents LGTM, just to make sure, the `./apisix` folder is created from where?

folder `./apisix` create by CMD `sudo mkdir -p apisix`, user can replace it with any other folder.

File: docs/en/latest/how-to-build.md [hunk 1]
```diff
@@ -54,6 +54,23 @@ $ sudo yum --showduplicates list apisix
...
 
+### Installation via RPM Offline Package(CentOS 7)
+
+Download APISIX offline RPM package to `./apisix` folder
+
+```shell
+$ sudo mkdir -p apisix
+$ sudo yum install -y https://repos.apiseven.com/packages/centos/apache-apisix-repo-1.0-1.noarch.rpm
+$ sudo yum clean all && yum makecache
+$ sudo yum install -y --downloadonly --downloaddir=./apisix apisix
```
Comment (leslie-tsang) [inline docs/en/latest/how-to-build.md:65]:
```suggestion
# Create a folder `apisix` to store the RPM packages
sudo mkdir -p apisix
# Install apisix repo
sudo yum install -y https://repos.apiseven.com/packages/centos/apache-apisix-repo-1.0-1.noarch.rpm
# Rebuild YUM cache
sudo yum clean all && yum makecache
# Download apisix and its deps package to the `apisx` folder
sudo yum install -y --downloadonly --downloaddir=./apisix apisix
```
Shall we remove `$` ? @juzhiyuan 
IMO, `$` is used to distinguish between `command` and its `output` like
```bash
$ uptime 
17:24:16 up 3 days,  8:06,  1 user,  load average: 0.00, 0.01, 0.05
```

File: docs/en/latest/how-to-build.md [hunk 2]
```diff
@@ -54,6 +54,23 @@ $ sudo yum --showduplicates list apisix
...
+```
+
+Copy `./apisix` folder to the target host, run the following command to install Apache APISIX.
+
+```shell
+$ yum install ./apisix/*.rpm
```
Comment (leslie-tsang) [inline docs/en/latest/how-to-build.md:71]: Shall we add a `sudo` here ?

File: docs/zh/latest/how-to-build.md [hunk 3]
```diff
@@ -54,6 +54,23 @@ $ sudo yum --showduplicates list apisix
...
 
+### 通过 RPM 包离线安装（CentOS 7）
+
+下载 APISIX 离线 RPM 包到 `./apisix` 文件夹
+
+```shell
+$ sudo mkdir -p apisix
+$ sudo yum install -y https://repos.apiseven.com/packages/centos/apache-apisix-repo-1.0-1.noarch.rpm
+$ sudo yum clean all && yum makecache
+$ sudo yum install -y --downloadonly --downloaddir=./apisix apisix
```
Comment (leslie-tsang) [inline docs/zh/latest/how-to-build.md:65]: Ditto

File: docs/zh/latest/how-to-build.md [hunk 4]
```diff
@@ -54,6 +54,23 @@ $ sudo yum --showduplicates list apisix
...
+```
+
+拷贝 `./apisix` 文件夹到目标主机，使用以下命令安装 Apache APISIX。
+
+```shell
+$ yum install ./apisix/*.rpm
```
Comment (leslie-tsang) [inline docs/zh/latest/how-to-build.md:71]: Ditto

**Review by leslie-tsang — CHANGES_REQUESTED** [review]

File: docs/en/latest/how-to-build.md (hunk 2 above)
Comment (123liubao) [reply docs/en/latest/how-to-build.md]: OK , I've already added this. Done.

File: docs/zh/latest/how-to-build.md (hunk 4 above)
Comment (123liubao) [reply docs/zh/latest/how-to-build.md]: OK , I've already added this. Done.

**Comment by juzhiyuan** [comment]
> > Contents LGTM, just to make sure, the `./apisix` folder is created from where?
> 
> @juzhiyuan I'm not sure what do you mean , `sudo mkdir -p apisix` this step is to create the folder.

Oh yes, I didn't notice this order 😅

image

**Review by leslie-tsang — APPROVED** [review]

File: docs/en/latest/how-to-build.md (hunk 1 above)
Comment (juzhiyuan) [reply docs/en/latest/how-to-build.md:65]: `uptime` is a command, right? 😄

Comment (leslie-tsang) [reply docs/en/latest/how-to-build.md:65]: yep

**Review by spacewander — APPROVED** [review]

**Pull request merged**